python-service/app/api/pptx.py

```
@@ -0,0 +1,241 @@
+
+        # Return translated file
+        return FileResponse(
+            path=str(temp_output_path),
+            filename="translated.pptx",
+            media_type="application/vnd.openxmlformats-officedocument.presentationml.presentation"
+        )
+
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"Translation application failed: {str(e)}", exc_info=True)
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)